mobile/src/main/java/ch/epfl/sdp/mobile/ui/social/SocialScreenState.kt

```diff
@@ -39,4 +39,11 @@ interface SocialScreenState {
 
   /** A callback invoked when the user type in the search text field */
   fun onValueChange()
+
+  /**
+   * A callback invoked when the user follows another user
+   *
+   * @param followed A [Person] that the current user wants to follow
+   */
+  fun onFollow(followed: Person)
```

Comment:
The convention would be to name this method `onFollowClick`, to keep it consistent with the rest of the codebase since `onFollow` does not communicate well enough the action taken by the user.